SECURITY.md

```
@@ -0,0 +1,2 @@
+- Please do not create GitHub issues to report security vulnerabilities.
+- Instead, report them to: support@clerk.dev
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)